### DISCUSSION TITLE
Import string>>?

### DISCUSSION THREAD

**Discussion by ALPOINTIntlGroup** [Opinionated-Blueprints/10-Books-Full-of-Rails#131] [Q&A] [answered]
Hi! Not entirely sure where the import string for this blueprint book is.. Anyone capable of assisting>? :)

**Comment by JonnyBro** [comment, marked as answer]
Open the [Releases](https://github.com/Opinionated-Blueprints/10-Books-Full-of-Rails/releases) tab and download the TXT file.

**Comment by jjMustang** [comment]
Thank you for the answer. Could that answer be added to the Readme section? I'm not familiar with GitHub, so I was unaware of the "releases" area.  I've always gotten the blueprint string from FactorioPrints, but I knew it had to be here somewhere.

**Reply by rpdelaney** [reply]
Not a bad idea.

**Reply by rpdelaney** [reply]
Done in b2615b069ab62f3febf8a310dbf11fbeae4c055a
